src/tools/CMakeLists.txt

```diff
@@ -48,7 +48,7 @@ set(mesh_tester_sources
     mesh_tester.cpp
 )
 set(mesh_tester_depends 
-    core slic mint slam quest primal cli11
+    axom
```

Comment:
This way, the mesh_tester depends on the whole axom library (which can be compiled as a dll) rather than the various components, which can't.